slidechain/export.go

```diff
@@ -90,12 +99,27 @@ func (c *Custodian) pegOutFromExports(ctx context.Context) {
 			}
 
 			log.Printf("pegging out export %x: %d of %s to %s", txid, amounts[i], asset.String(), exporters[i])
-			// TODO(vniu): flag txs that fail with unretriable errors in the db
+
+			var peggedOut pegOutState
```

Comment:
nit: could be slightly more readable to initialize to `peggedOut = pegOutOK` and then re-assign `peggedOut` on failure cases